robottelo/vm.py

```diff
@@ -365,6 +365,12 @@ def enable_repo(self, repo, force=False):
         if force or settings.cdn or not downstream_repo:
             self.run(u'subscription-manager repos --enable {0}'.format(repo))
 
+    def list_repos(self):
+        return self.run("subscription-manager repos --list")
+
+    def status(self):
```

Comment:
This method name could be more descriptive. Looking at the name, a user isn't sure of what kind of status they are getting. maybe ```sub_status``` or ```subscription_status```?

Reply:
Okay,  I will do that, 
just this is vm module, so user can expect subscription manager,
but in future there can be another status, 
but still when you look at function, what I always do, I can see the only line where is the `subscription-manager status`